website/docs/en/config/tools/bundler-chain.mdx

```diff
@@ -141,6 +141,28 @@ export default {
 };
 ```
 
+### rspack
+
+- **Type:** `Rspack`
+
+The Rspack instance. For example:
+
+```js
+export default {
+  tools: {
+    bundlerChain: (chain, { rspack }) => {
+      chain.plugin('define').use(rspack.DefinePlugin, [
```

Comment:
The `'define'` id is the same as the Rsbuild builtin `CHAIN_ID.PLUGIN.DEFINE`, can we use `extra-define` in this example to avoid conflict.